content/news/2016/04/2016-04-06-remixing-content-shining-sunlight-on-the-sunshot-initiatives-solar-energy-rd-project-map.md

```diff
@@ -6,8 +6,6 @@ summary: 'With more than 400 projects in their portfolio, it can be difficult fo
 authors:
   - susanna-murley
 topics:
-  - data
-  - monthly-theme
   - DOE
```

Comment:
```suggestion
  - product-and-project-management
```

Reply:
Use `open-data` and `data-visualization`